src/config.rs

```diff
@@ -243,6 +243,7 @@ pub struct Python {
     enable_pip_review: Option<bool>,
     enable_pip_review_local: Option<bool>,
     enable_pipupgrade: Option<bool>,
+    pipupgrade_arguments: Option<String>,
```

Comment:
Since this PR changes the configuration, would you like to give [config.example.toml](https://github.com/topgrade-rs/topgrade/blob/f6e8af186c76e742784557d6cf324377f715f5fd/config.example.toml#L112-L115) an update to include this change:)

Reply:
Done. Sorry I missed that. Thank you!

Reply:
Thanks! :)